### ISSUE TITLE
Exception when I'm trying create Post without thumbnail

### ISSUE THREAD

**Issue by aivus** [Madlexx/symfonyApplication#8]
```
Catchable Fatal Error: Argument 1 passed to Application\BlogBundle\Service\Thumbnail::save() must be an instance of Symfony\Component\HttpFoundation\File\UploadedFile, null given, called in /Users/aivus/projects/alexeySymfonyApp/src/Application/BlogBundle/Form/DataTransformer/UploadedFileToThumbnail.php on line 31 and defined
```

**Comment by Madlexx** [comment]
Fixed.

**Comment by aivus** [comment]
:ballot_box_with_check:

**Comment by aivus** [comment]
Still getting this on `fixes` branch

**Comment by Madlexx** [comment]
Fixed #8
